lib/src/jira/workflow.rs

```diff
@@ -339,18 +342,23 @@ pub async fn merge_pending_versions(
     }
 
     // create the target version for this project
-    if real_versions.iter().any(|v| v.name == version) {
-        info!(
-            "JIRA version {} already exists for project {}",
-            version, project
-        );
-    } else {
-        info!(
-            "Creating new JIRA version {} for project {}",
-            version, project
-        );
-        jira.add_version(project, version).await?;
-    }
+    let id = match real_versions.into_iter().find(|v| v.name == version) {
+        Some(v) => {
+            info!(
+                "JIRA version {} already exists for project {}",
+                version, project
+            );
+            v.id
```

Comment:
feels odd to return the id in this case, since we're not creating it -- are you thinking you'd show a link to the version even if we don't create it or list any versions we're adding to it?

Reply:
yes, if we merge pending versions to an existing fix version, I think it still makes sense to post the link.
The success text doesn't explicitly say it was newly created, so I think it makes sense either way.

If you try to merge to an existing fix version and there are no relevant pending versions for it, then the submit button is disabled, so we won't ever show a link in that case.